docs/subnets/customize-a-subnet.md

```diff
@@ -505,6 +507,122 @@ It's possible to enable this precompile with an initial configuration to activat
 
 This will set the fee config to the values specified in the `initialFeeConfig` field. For further information about precompile initial configurations see [Initial Precompile Configurations](#initial-precompile-configurations).
 
+## Changing Fee Reward Mechanisms
```

Comment:
Should this be in the same level as other Precompiles such as `Configuring Dynamic Fees`? if so, it should start with `###` instead of `##`
```suggestion
### Changing Fee Reward Mechanisms
```

Reply:
good catch!